File: PRAK604-2210817220001-AjengDiahPramesti.py
kode1=str(input())
kode2 =str(input())
kata1 = len(kode1) 
kata2 = len(kode2)
a= 0 
b = 0

if kata1 == kata2 :
    for i in range (0, kata1):
        if kode1 [i]== kode2 [i]:
            if kode1 [i]==' ':
                print(" ", end=' ')
            else: 
                print("*", end=' '); a+=1
        else:
            print("#", end=' '); b+=1    
    print("\n*= ", a)
    print("#= ", b)
    if a >= b:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")
else:
    print("Panjang kalimat berbeda, pesan palsu")     

File: PRAK604-2210817220001-AjengDiahPramesti.c
#include <stdio.h>
#include <string.h>

int main()
{
    int a, b;
    char kode1[50], kode2[50];
    printf(""); gets(kode1);
    printf(""); gets(kode2);
    int kata1=strlen(kode1), kata2=strlen(kode2);
    if(kata1 != kata2){
        printf("Panjang kalimat berbeda, pesan palsu");
    }
    else{
        int a=0, b=0 ;
        for (int c=0; c<kata1; c++){
            if (kode2[c]==' ' && kode1[c]==' '){
                continue;
            }
            else if(kode1[c] != kode2[c]){
                printf("#");
                a++;
            }
            else if(kode1[c] == kode2[c]){
                printf("*");
                b++;
            }
        }
        printf("\n*= %d", b);
        printf("\n#= %d", a);
        if (b >= a){
            printf("\nPesan Asli");}
        else{
            printf("\nPesan Palsu");}
    }
}